core/src/main/java/org/mule/retry/policies/AbstractPolicyTemplate.java

```diff
@@ -37,6 +38,8 @@
 
     private MuleContext muleContext;
 
+    private AtomicBoolean cancelStart = new AtomicBoolean(false);
```

Comment:
Shouldn't this be also named stopRetrying?

Reply:
Applied the suggested changes